orchestrator/workflows/tasks/resume_workflows.py

```diff
@@ -46,14 +68,44 @@ def resume_found_workflows(waiting_process_ids: list[UUIDstr]) -> State:
             processes.resume_process(process)
             resumed_process_ids.append(process_id)
         except Exception:
-            logger.exception()
+            logger.warning()
+        finally:
+            # Make sure to turn it on again
+            db.session.info["disabled"] = True
+
+    return {
+        "number_of_resumed_process_ids": len(resumed_process_ids),
+        "resumed_process_ids": resumed_process_ids,
+    }
+
+
+@step("Restart found CREATED workflows")
+def restart_created_workflows(created_state_process_ids: list[UUIDstr]) -> State:
+    started_process_ids = []
+    for process_id in created_state_process_ids:
+        try:
+            process = db.session.get(ProcessTable, process_id)
+            if not process:
+                continue
+            # Workaround the commit disable function
+            db.session.info["disabled"] = False
+            processes.restart_process(process)
+            started_process_ids.append(process_id)
+        except Exception:
+            logger.warning()
```

Comment:
```suggestion
            logger.warning("Could not restart process", process_id=process_id, error=str(exc))
```

and change `        except Exception as exc:` above